example/metric/exponential_histogram/example.go

```diff
@@ -0,0 +1,322 @@
+		sdkmetric.Instrument{
+			Name: "latency",
+		},
+		sdkmetric.Stream{
+			Name: "latency_c",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: defaultBuckets,
+			},
+		},
+	)
+
+	viewLatencyShiftedA := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_shifted",
+		},
+		sdkmetric.Stream{
+			Name: "latency_shifted_a",
+			Aggregation: sdkmetric.AggregationBase2ExponentialHistogram{
+				MaxSize:  160,
+				MaxScale: 20,
+			},
+		},
+	)
+
+	viewLatencyShiftedB := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_shifted",
+		},
+		sdkmetric.Stream{
+			Name: "latency_shifted_b",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: linearBuckets,
+			},
+		},
+	)
+
+	viewLatencyShiftedC := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_shifted",
+		},
+		sdkmetric.Stream{
+			Name: "latency_shifted_c",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: defaultBuckets,
+			},
+		},
+	)
+
+	viewLatencyMultimodalA := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_multimodal",
+		},
+		sdkmetric.Stream{
+			Name: "latency_multimodal_a",
+			Aggregation: sdkmetric.AggregationBase2ExponentialHistogram{
+				MaxSize:  160,
+				MaxScale: 20,
+			},
+		},
+	)
+
+	viewLatencyMultimodalB := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_multimodal",
+		},
+		sdkmetric.Stream{
+			Name: "latency_multimodal_b",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: linearBuckets,
+			},
+		},
+	)
+
+	viewLatencyMultimodalC := sdkmetric.NewView(
+		sdkmetric.Instrument{
+			Name: "latency_multimodal",
+		},
+		sdkmetric.Stream{
+			Name: "latency_multimodal_c",
+			Aggregation: sdkmetric.AggregationExplicitBucketHistogram{
+				Boundaries: defaultBuckets,
+			},
+		},
+	)
+
+	// initialize a MeterProvider with that periodically exports to the GCP exporter.
+	provider := sdkmetric.NewMeterProvider(
+		sdkmetric.WithView(viewLatencyA),
+		sdkmetric.WithView(viewLatencyB),
+		sdkmetric.WithView(viewLatencyC),
+		sdkmetric.WithView(viewLatencyShiftedA),
+		sdkmetric.WithView(viewLatencyShiftedB),
+		sdkmetric.WithView(viewLatencyShiftedC),
+		sdkmetric.WithView(viewLatencyMultimodalA),
+		sdkmetric.WithView(viewLatencyMultimodalB),
+		sdkmetric.WithView(viewLatencyMultimodalC),
+		sdkmetric.WithReader(
+			sdkmetric.NewPeriodicReader(
+				exporter,
+				sdkmetric.WithInterval(10*time.Second),
+			),
+		),
+		sdkmetric.WithResource(res),
+	)
+	defer func() {
+		if err = provider.Shutdown(ctx); err != nil {
+			log.Fatalf("failed to shutdown meter provider: %v", err)
+		}
+	}()
+
+	// Create a meter with which we will record metrics for our package.
+	meter := provider.Meter("github.com/GoogleCloudPlatform/opentelemetry-operations-go/example/metric")
+
+	_, err = meter.Float64ObservableGauge(
+		"latency",
+		metric.WithDescription(
+			"Latency",
+		),
+		metric.WithUnit("ms"),
```

Comment:
nit: prefer using seconds for durations in samples

Reply:
Is there a functionality difference between using `s` or `ms` ? 

For this specific example, i chose `ms` since the generated data ("Latency" Log-Normal Distribution) has a mean of around ~ `30`. So, I think it seems more realistic `30ms` of server delay over `30s`. Though, this is purely an example, so at the end there is no real difference.